dotnet_sdk/lib/dependabot/dotnet_sdk/metadata_finder.rb

```diff
@@ -0,0 +1,23 @@
+# typed: strong
+# frozen_string_literal: true
+
+require "dependabot/metadata_finders"
+require "dependabot/metadata_finders/base"
+
+module Dependabot
+  module DotnetSdk
+    class MetadataFinder < Dependabot::MetadataFinders::Base
+      extend T::Sig
+
+      private
+
+      sig { override.returns(T.nilable(Dependabot::Source)) }
+      def look_up_source
+        # TODO: Where to link to?
```

Comment:
SDK versions commonly have a tag like here: `https://github.com/dotnet/sdk/tree/v8.0.400` would that be useful?

Reply:
Absolutely!

Reply:
Turns out `Dependabot::Source` trims everything after the repo name, so a static value of `https://github.com/dotnet/sdk` is the right answer here.